problems/strings.js

```diff
@@ -68,7 +82,13 @@ verbing('go')
  
  */
 
-function verbing() {}
+function verbing(str) {
+  if (str.slice(-3) === "ing") {
+    return str + "ly";
+  } else if (str.length >= 3) {
+    return str + "ing";
+  } else return str;
```

Comment:
Remember to add the curly braces {} after the else statement for the return str code block.